### ISSUE TITLE
EMG_dataanalysis

### ISSUE THREAD

**Issue by ssperry42** [spyder-ide/spyder#21078]
### What steps will reproduce the problem?

upload 3 data files, 10 50 and 90 percentiles of the emg data 

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 164, in lock
    symlink(str(os.getpid()), self.name)
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 96, in symlink
    rename(newlinkname, filename)
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\sophi\\.spyder-py3\\spyder.lock.1687457687233.newlink' -> 'C:\\Users\\sophi\\.spyder-py3\\spyder.lock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 191, in lock
    kill(int(pid), 0)
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 80, in kill
    raise OSError(errno.ESRCH, None)
ProcessLookupError: [Errno 3] None

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\app\start.py", line 219, in main
    lock_created = lock.lock()
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 199, in lock
    rmlink(self.name)
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\utils\external\lockfile.py", line 121, in rmlink
    os.remove(os.path.join(filename, 'symlink'))
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\sophi\\.spyder-py3\\spyder.lock\\symlink'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\plugins\editor\plugin.py", line 2434, in save_as
    if editorstack.save_as():
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\plugins\editor\widgets\editor.py", line 2087, in save_as
    new_index = self.rename_in_data(original_filename,
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\plugins\editor\widgets\editor.py", line 1376, in rename_in_data
    finfo.editor.document_did_open()
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\plugins\completion\decorators.py", line 20, in wrapper
    params = req(self, *args, **kwargs)
  File "C:\Users\sophi\Documents\Anaconda\lib\site-packages\spyder\plugins\editor\widgets\codeeditor.py", line 1201, in document_did_open
    self._timer_sync_symbols_and_folding.timeout.connect(
TypeError: connection is not unique
```

## Versions

* Spyder version: 5.4.1  (conda)
* Python version: 3.10.9 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<9.0.0       :  8.10.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.17.3 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.5.4 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<3.0           :  2.16.2 (OK)
pylint_venv >=2.1.1           :  2.3.0 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.7.0;<1.8.0          :  1.7.1 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.2 (OK)
qstylizer >=0.2.2             :  0.2.2 (OK)
qtawesome >=1.2.1             :  1.2.2 (OK)
qtconsole >=5.4.0;<5.5.0      :  5.4.0 (OK)
qtpy >=2.1.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  65.6.3 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.4.1;<2.5.0 :  2.4.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=22.1.0                  :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  3.7.0 (OK)
numpy >=1.7                   :  1.23.5 (OK)
pandas >=1.1.1                :  1.5.3 (OK)
scipy >=0.17.0                :  1.10.0 (OK)
sympy >=0.7.3                 :  1.11.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @ssperry42, thanks for reporting. I think this problem is fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
